ipcdefs/audio.id

```diff
@@ -1,3 +1,38 @@
+type nn::audio::detail::AudioRendererParameterInternal = struct {
```

Comment:
`struct` can be specialized as `struct<size>` to make it clear to any implementations what the size of the struct is intended to be